MdeModulePkg/Core/DxeIplPeim/Ia32/DxeLoadFunc.c

```diff
@@ -461,6 +474,9 @@ HandOffToDxeCore (
       BaseOfStack,
       STACK_SIZE
       ));
+    if (PrepareForUniversalPayloadPpi) {
+      PrepareForUniversalPayloadPpi->PrepareForUniversalPayload ();

```

Comment:
Considering the future CBOR, how about code as below?
VOID *UplInterface = PrepareForUniversalPayloadPpi->PrepareForUniversalPayload (HobList.Raw);

Reply:
hi Ray, 
how about the code below?

VOID *Interface;
if (PrepareForUniversalPayloadPpi != NULL){
  Status = PrepareForUniversalPayloadPpi->PrepareForUniversalPayload (HobList.Raw, &Interface);
  if (RETURN_ERROR(Status)){
     // Should not get here
     CpuDeadLoop ();
  }
} else {
  Interface = HobList.Raw;
}